packages/perseus/src/widgets/radio/__stories__/choice.stories.jsx

```diff
@@ -0,0 +1,135 @@
+// @flow
+import * as React from "react";
+
+import Choice from "../choice.jsx";
+
+type StoryArgs = {|
+    checked: boolean,
+    rationale: React.Node,
+    content: React.Node,
+    correct: boolean,
+    disabled: boolean,
+    pos: number,
+    reviewMode: boolean,
+    showRationale: boolean,
+    showCorrectness: boolean,
+    multipleSelect: boolean,
+    crossedOut: boolean,
+    previouslyAnswered: boolean,
+|};
```

Comment:
**suggestion:** I think it would be good to include `onChange: () => {},` here as well.